src/handler/handlers.jl

```diff
@@ -190,7 +232,12 @@ function make_handler(app::DashApp, registry::ResourcesRegistry; check_layout =
     handler = state_handler(router, state)
     get_setting(app, :compress) && (handler = compress_handler(handler))
 
-    HTTP.handle(handler, HTTP.Request("GET", prefix)) #For handler precompilation
+    compile_request = HTTP.Request("GET", "/test_big")
```

Comment:
@waralex Is this request for `/test_big` just to create the object `compile_request` used to precompile the handler in the line below?